src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.header1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.header2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+.href {
+  color: green;
+}
+
+.parag {
+  margin: 0;
+}
+
+.header1 {
+  margin: 0;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.